components/server/src/ome/security/KeyAndTrustStoreConfiguration.java

```diff
@@ -42,9 +42,17 @@ public void afterPropertiesSet() throws Exception {
                 log.warn("Overwriting existing trust store: " + oldTrustStore);
             }
         }
-        System.setProperty(JAVAX_NET_SSL_TRUST_STORE, trustStore);
-        System.setProperty(JAVAX_NET_SSL_TRUST_STORE_PASSWORD,
-                trustStorePassword);
+        if (trustStore == null || trustStore.length() < 1) {
```

Comment:
note strings also have an `isEmpty()` and there're even utils like null-safe `StringUtils.isBlank`

Reply:
Thanks, I've changed it to use `StringUtils.isEmpty` so we've still go the option of using `[space]` to distinguish between empty and unset/default.